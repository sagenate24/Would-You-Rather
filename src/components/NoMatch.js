import React from 'react';

const NoMatch = () => (
      <div>
        <h1>404:</h1>
        <h3>Page not found</h3>
      </div>
);

export default NoMatch;
