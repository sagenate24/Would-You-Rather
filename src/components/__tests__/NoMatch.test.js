import React from 'react';
import * as renderer from 'react-test-renderer';
import NoMatch from '../NoMatch';

it('should render correctly', () => {

  const tree = renderer.create(
    <NoMatch />
  ).toJSON();
  expect(tree).toMatchSnapshot();

});