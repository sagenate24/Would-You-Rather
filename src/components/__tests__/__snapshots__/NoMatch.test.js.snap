// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <h1>
    404:
  </h1>
  <h3>
    Page not found
  </h3>
  <br />
  <img
    alt="sad face"
    src={
      Object {
        "0": "s",
        "1": "a",
        "10": "g",
        "2": "d",
        "3": "F",
        "4": "a",
        "5": "c",
        "6": "e",
        "7": ".",
        "8": "p",
        "9": "n",
        "default": "sadFace.png",
      }
    }
  />
</div>
`;
